garbage-collection-configuration.md

```diff
@@ -1,134 +1,18 @@
+Garbage collection is configured via the following system variables:
 
-{{< copyable "sql" >}}
-
-```sql
-select VARIABLE_NAME, VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME like "tikv_gc%";
-```
-
-```sql
-+--------------------------+----------------------------------------------------------------------------------------------------+
-| VARIABLE_NAME            | VARIABLE_VALUE                                                                                     |
-+--------------------------+----------------------------------------------------------------------------------------------------+
-| tikv_gc_leader_uuid      | 5afd54a0ea40005                                                                                    |
-| tikv_gc_leader_desc      | host:tidb-cluster-tidb-0, pid:215, start at 2019-07-15 11:09:14.029668932 +0000 UTC m=+0.463731223 |
-| tikv_gc_leader_lease     | 20190715-12:12:14 +0000                                                                            |
-| tikv_gc_enable           | true                                                                                               |
-| tikv_gc_run_interval     | 10m0s                                                                                              |
-| tikv_gc_life_time        | 10m0s                                                                                              |
-| tikv_gc_last_run_time    | 20190715-12:09:14 +0000                                                                            |
-| tikv_gc_safe_point       | 20190715-11:59:14 +0000                                                                            |
-| tikv_gc_auto_concurrency | true                                                                                               |
-| tikv_gc_mode             | distributed                                                                                        |
-+--------------------------+----------------------------------------------------------------------------------------------------+
-13 rows in set (0.00 sec)
-```
-
-For example, the following statement makes GC keep history data for the most recent 24 hours:
-
-```sql
-update mysql.tidb set VARIABLE_VALUE="24h" where VARIABLE_NAME="tikv_gc_life_time";
-```
-
-> **Note:**
->
-> In addition to the following GC configuration parameters, the `mysql.tidb` system table also contains records that store the status of the storage components in a TiDB cluster, among which GC related ones are included, as listed below:
->
-> - `tikv_gc_leader_uuid`, `tikv_gc_leader_desc` and `tikv_gc_leader_lease`: Records the information of the GC leader
-> - `tikv_gc_last_run_time`: The duration of the latest GC (updated at the beginning of each round of GC)
-> - `tikv_gc_safe_point`: The current safe point (updated at the beginning of each round of GC)
```

Comment:
The explanation for the non-config fields can be kept

Reply:
I removed them because the topic of the page is configuration. These are status variables, and not intended to be maintainable.  I have a separate PR for them as well https://github.com/pingcap/tidb/pull/22286